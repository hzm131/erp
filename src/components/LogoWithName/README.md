LogoWithName example:

```js
<LogoWithName fill="#242424" width={200} />
```
