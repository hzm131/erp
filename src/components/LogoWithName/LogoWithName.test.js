import React from 'react'
import { mount } from 'enzyme'
import LogoWithName from './LogoWithName'

it('should render using mount', ()=>{
  const mounted = mount(<LogoWithName width={50} fill="black"/>)

  expect(mounted).toMatchSnapshot()
})
