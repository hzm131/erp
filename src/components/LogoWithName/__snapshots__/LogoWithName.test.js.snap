// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render using mount 1`] = `
<svg
  viewBox="0 0 545.11 89.05"
  width={50}
  xmlns="http://www.w3.org/2000/svg"
>
  <defs>
    <style>
      .cls-1,.cls-2{fill:black;}.cls-1{fill-rule:evenodd;}
    </style>
  </defs>
  <path
    className="cls-1"
    d="M178.1,60.15a9.56,9.56,0,1,0-9.56-9.55A9.57,9.57,0,0,0,178.1,60.15Z"
  />
  <path
    className="cls-1"
    d="M203.32,81.62V74.71h17.17v6.91Zm251.5-5.7h2.54v7h-2.62A21.24,21.24,0,0,1,445,80.16a26.34,26.34,0,0,1-7.61-7.64L426.06,55.77H456v6.92H440l5.75,8.4A12.23,12.23,0,0,0,450,75a12.6,12.6,0,0,0,4.79.94ZM203.32,53.21V46.29h17.17v6.92Zm0-25.53V20.76h17.17v6.92Zm22.39-1.53v-6.3h56.84v6.3H225.71Zm52.83,3.75V71.61q0,10.92-12.81,10.93H249.84V76.15h14.61c2.36,0,3.93-.48,4.69-1.44.6-.72.91-2.1.91-4.16V29.9ZM227.33,62.62q0,4.65,2.55,6.41c1.28.91,3.65,1.35,7.08,1.35h18.63c3.43,0,5.79-.7,7.06-2.08s1.8-3.34,1.8-6.17V40.22q0-4.36-2.51-6.26c-1.49-1.11-3.87-1.65-7.17-1.65H236.49c-3.15,0-5.5.67-7,2s-2.13,3.23-2.13,6V62.62Zm11.26,1h14.46a6.69,6.69,0,0,0,2.79-.34,2.46,2.46,0,0,0,.73-2.13v-19a3.27,3.27,0,0,0-.55-2.26,4.09,4.09,0,0,0-2.8-.67H238.59a4.54,4.54,0,0,0-3,.67,4.24,4.24,0,0,0-.42,2.44V60.84a3.25,3.25,0,0,0,.55,2.22,4.65,4.65,0,0,0,2.83.56Zm181-10.06h30.13c5.11,0,7.65-2.25,7.65-6.69l0-20.88a5.31,5.31,0,0,0-2.08-4.27,9.31,9.31,0,0,0-6-1.74l-30.35,0a7.39,7.39,0,0,0-5.13,1.7A6,6,0,0,0,412,23.85a10.2,10.2,0,0,0-.39,3.28V75.52c0,2.42.66,4.18,2,5.3,1.47,1.25,3.87,1.87,7.23,1.87h10V75.88H423.4a6.13,6.13,0,0,1-3.2-.54,2,2,0,0,1-.64-1.63l0-20.15Zm0-6.89V40.44h29.87l.09,4.24a1.88,1.88,0,0,1-.64,1.51,4.44,4.44,0,0,1-2.53.48Zm0-13.15L419.52,29c0-1.45,1.05-2.19,3.24-2.19H446.7c1.91,0,2.86.76,2.86,2.27l-.09,4.49ZM378.77,75V27.24c0-2.24.51-3.83,1.53-4.8a7.92,7.92,0,0,1,5.6-1.91h10.87c3,0,5.17.64,6.42,1.91,1.08,1.09,1.63,2.82,1.63,5.17V74.94q0,6.77-7.58,6.76H385.06q-6.29,0-6.29-6.66Zm18.64-12.18H386V73.71c0,1.24.81,1.86,2.4,1.86l6.93,0c1.4,0,2.1-.67,2.1-2l0-10.67ZM386,56.24h11.38v-10H386v10Zm0-16.56h11.38l0-10.54q0-2.19-2.1-2.19h-7a2.41,2.41,0,0,0-2,.7,3.27,3.27,0,0,0-.34,1.78V39.68Zm80,22.57V55.41h8.47V46.59H466V39.68h27.93v6.91H482.33v8.82h11.58v6.84H482.33V72.72A3.14,3.14,0,0,0,483,75.1a3.34,3.34,0,0,0,2.27.67h8.63v6.84H482a8.05,8.05,0,0,1-5.37-1.75,6.46,6.46,0,0,1-2.17-5.28V62.25Zm0-24.88V30.49a5.5,5.5,0,0,0,3.13-1.73c.6-.78,1.75-3.71,3.46-8.76H481l-1.37,4.1h14.27V31H477.26a11.44,11.44,0,0,1-4.74,4.67A16.79,16.79,0,0,1,466,37.37Zm29.94,5.35V36.64h5.34V32.27h7.91v4.37h21.89V32.27h8v4.37H545v6.08Zm0-13.6V23.27h20.65v-3.5h7.89v3.5H545v5.85ZM532.63,70v4.18c0,1.49,1,2.24,3.12,2.24h9.36V82.9H533.27A10.3,10.3,0,0,1,527,81.15a5.89,5.89,0,0,1-2.77-5V70h-6.53a25.24,25.24,0,0,1-19.49,12.65v-6.8a17.93,17.93,0,0,0,10-5.85h-3.6c-2.78,0-4.82-.63-6-1.89Q497,66.61,497,63.2V51.31a5.42,5.42,0,0,1,2-4.28c1.08-1,2.82-1.43,5.25-1.43H536c2.86,0,4.86.54,6,1.6a6.5,6.5,0,0,1,2,5.11V62.1Q544,70,535.75,70Zm3.26-9.69H509.26V55.18h26.63V53.5c0-1.61-.94-2.43-2.79-2.43H508.35c-2.07,0-3.13.58-3.16,1.72l0,1.54h0v7.4a2.37,2.37,0,0,0,.93,2.24,5.51,5.51,0,0,0,2.61.41h24.18a3.39,3.39,0,0,0,2.32-.6,2.62,2.62,0,0,0,.69-2.05Z"
  />
  <path
    className="cls-1"
    d="M295.19,31h7.91L302,40.71c-.09.93-.13,1.71-.13,2.38A3.51,3.51,0,0,0,303,45.82,5.46,5.46,0,0,0,306.78,47h50.73c4,0,6.9.8,8.78,2.38a8.15,8.15,0,0,1,3.06,6.45V72.58a9.26,9.26,0,0,1-3.06,7.29,11.78,11.78,0,0,1-7.93,2.57H295.71V75.53h60.81c3.37,0,5-1.75,5-5.23V58.1q0-4.14-4.49-4.14h-51c-3.13,0-5.47-.41-7-1.23Q294,50,294,44.94a6.46,6.46,0,0,1,0-.74Z"
  />
  <path
    className="cls-1"
    d="M292.63,26.68V19.92h61a12.26,12.26,0,0,1,8.42,2.79,7.13,7.13,0,0,1,2.72,5.61c0,1-.07,2.16-.2,3.45L363.42,43.3h-7.88l1.3-11.37a14.32,14.32,0,0,0,.09-1.45,3.07,3.07,0,0,0-.85-2.27,5.08,5.08,0,0,0-3.88-1.53Z"
  />
  <rect
    className="cls-2"
    height="6.91"
    width="65.73"
    x="290.63"
    y="61.32"
  />
  <path
    className="cls-1"
    d="M34,87.23a50.16,50.16,0,0,1-22.11-13A42.26,42.26,0,0,1,.37,49.92C-1.78,33.41,5.76,12.56,13.52,0,24.39,16.25,57.13,43.65,87,58.85A115.79,115.79,0,0,1,73.65,64.1C52.89,54.46,32.09,40.33,17.2,25.18c-3.41,10-4.93,18.23-3.34,25,5,21.11,27.5,25.45,44.52,23,13.4-2,28.91-8.2,40.44-14.53h0c26.57-14.29,43.43-33.33,46.44-37.71,17.16,50-8,61.65-42.32,53.65,3.22-1.83,6.46-3.8,9.57-5.89,19.2,4.23,32.68-2.15,30.33-28.74C118.7,62.79,68.69,97.41,34,87.23Z"
  />
</svg>
`;
